### ISSUE TITLE
didn't converge

### ISSUE THREAD

**Issue by zhengduoru** [jfzhang95/pytorch-deeplab-xception#1]
hi, I tried your code without loading pretrained model.  But I found it didn't converge at all. What should I do to make it work. 

[Epoch: 0, numImages:  1464]
Loss: 754343.768581
Execution time: 61.669747280015144

[Epoch: 1, numImages:  1464]
Loss: 754388.596284
Execution time: 44.01166869502049

[Epoch: 2, numImages:  1464]
Loss: 753789.062500
Execution time: 44.40118463395629

[Epoch: 3, numImages:  1464]
Loss: 754601.996622
Execution time: 43.993270942009985

[Epoch: 4, numImages:  1464]
Loss: 754301.511824
Execution time: 44.05079116899287

[Epoch: 5, numImages:  1464]
Loss: 754124.653716
Execution time: 44.40390331699746

[Epoch: 6, numImages:  1464]
Loss: 753820.405405
Execution time: 44.04822350398172

[Epoch: 7, numImages:  1464]
Loss: 754613.310811
Execution time: 44.43823240103666

[Epoch: 8, numImages:  1464]
Loss: 754287.202703
Execution time: 44.060883720987476

[Epoch: 9, numImages:  1464]
Loss: 754111.920608
Execution time: 44.11198482802138

[Epoch: 9, numImages:  1449]
Loss: 756997.688530
[Epoch: 10, numImages:  1464]
Loss: 754386.648649
Execution time: 44.27616982697509

[Epoch: 11, numImages:  1464]
Loss: 754753.653716
Execution time: 44.208685109973885

[Epoch: 12, numImages:  1464]
Loss: 753667.771959
Execution time: 44.36111898702802

[Epoch: 13, numImages:  1464]
Loss: 754298.302365
Execution time: 44.42224766005529

[Epoch: 14, numImages:  1464]
Loss: 754134.364865
Execution time: 44.61775282601593

[Epoch: 15, numImages:  1464]
Loss: 754144.538851
Execution time: 44.4529153269832

[Epoch: 16, numImages:  1464]
Loss: 754300.746622
Execution time: 44.196999565989245

[Epoch: 17, numImages:  1464]
Loss: 753685.072635
Execution time: 44.14125015295576

**Comment by zhengduoru** [comment]
I print the loss.grad and I got None.
Do you know what's wrong in the loss.backward()?
